src/components/pages/Publications/PublicationsConstants.ts

```diff
@@ -508,31 +511,49 @@ export const listOfPrivateSectorReports: PublicationProps[] = [
     {
         titleKey1: 'IndustryReportTitles',
         titleKey2: ["OctoberReport"],
+        collection: "A Pillai, N Duarte, and M Rocco.",
```

Comment:
Will need to translate all the text here (I think only "and" needs to be translated lol)